app/views/admin/events/show-partials/_gallery.html.erb

```diff
@@ -0,0 +1,58 @@
+<section id="gallery" class="section-container">
+  <div class="row">
+    <div class="title">
+      Our Gallery
+    </div>
+    <div class="content">
+      <div class="row no-gutters">
+        <div class="col-lg-3 col-md-4">
+          <div class="photo-gallery">
+            <%= link_to image_tag("gallery/1.jpg", class: "img-fluid"), image_url("1.jpg"), class: "venobox", data: { gall: "photo-gallery" } %>
```

Comment:
can change this in the original views/events/show-partials/... 

Also, what will happen if someone insert two files with same name to the database? will there be any identifier or anything?

Reply:
What do you mean by "can change this in the original views/events/show-partials/..."?

For the second question, should be fine. From what I know, everytime we add a new image with active storage, it will be stored in a blob table and the blob table also have a primary key.

Reply:
I mean you use image_url right for it

Reply:
Yes

Reply:
Okok nvm, I will adjust the changes needed in the events view later